test/Table_Tests/src/Database/SQLite_Spec_New.enso

```diff
@@ -0,0 +1,145 @@
+from Standard.Base import all
+import Standard.Base.Runtime.Ref.Ref
+from Standard.Base.Runtime import assert
+import Standard.Base.Errors.File_Error.File_Error
+import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
+
+import Standard.Table.Data.Type.Value_Type.Bits
+from Standard.Table import Table, Value_Type
+from Standard.Table.Errors import Invalid_Column_Names, Duplicate_Output_Column_Names
+
+import Standard.Database.Data.Column.Column
+import Standard.Database.Internal.Replace_Params.Replace_Params
+from Standard.Database import all
+from Standard.Database.Errors import SQL_Error, Unsupported_Database_Operation
+
+from Standard.Test_New import all
+
+import project.Database.Common.Common_Spec_New
+import project.Database.Helpers.Name_Generator
+
+##
+    sqlite_specific_spec suite_builder prefix connection setup =
+        table_builder = setup.table_builder
+
+        suite_builder.group prefix+"Schemas and Databases" group_builder->
+            group_builder.specify "should be able to get current database and list databases" <|
+                connection.database . should_equal Nothing
+                connection.databases . should_equal [Nothing]
+                Meta.is_same_object connection (connection.set_database Nothing) . should_be_true
+
+            group_builder.specify "should be able to get current schema and list schemas" <|
+                connection.schema . should_equal Nothing
+                connection.schemas . should_equal [Nothing]
+                Meta.is_same_object connection (connection.set_schema Nothing) . should_be_true
+
+            group_builder.specify "does not allow changing schema or database" <|
+                connection.set_schema "foo" . should_fail_with SQL_Error
+                connection.set_database "foo" . should_fail_with SQL_Error
+
+        Test.group prefix+"Tables and Table Types" <|
```

Comment:
The code below is using the old `Test` API, I assume.

Reply:
Yes, its commented. It should have been removed. See my [other comment](https://github.com/enso-org/enso/pull/8622/files#r1452500371).